Parsers/ASimNetworkSession/Tests/SonicWall_SonicOS_ASimNetworkSession_DataTest.csv

```diff
@@ -0,0 +1,29 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 1000 records (100.0%) for field [DvcIpAddr] of type [IP Address]: [""18C24135E8B6""] (Schema:NetworkSession)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 1000 records (100.0%) for field [EventProduct] of type [Enumerated]: [""SonicOS""] (Schema:NetworkSession)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 931 records (93.1%) for field [DvcAction] of type [Enumerated]: [""NA""] (Schema:NetworkSession)"
+"(0) Error: 2 invalid value(s) (up to 10 listed) in 2000 records (200.0%) for field [EventVendor] of type [Enumerated]: [""SonicWall""] (Schema:NetworkSession)"

```

Comment:
Please add this to supported values in EventVendor for NetworkSession: https://github.com/Azure/Azure-Sentinel/blob/master/ASIM/dev/ASimTester/ASimTester.csv